Documentation/kubernetes.md

```diff
@@ -43,6 +43,7 @@ Additional notes:
 * Kubernetes only trusts ID Tokens issued to a single client.
   * As a work around dex allows clients to [trust other clients][trusted-peers] to mint tokens on their behalf.
 * If a claim other than "email" is used for username, for example "sub", it will be prefixed by `"(value of --oidc-issuer-url)#"`. This is to namespace user controlled claims which may be used for privilege escalation.
+* The `/etc/ssl/certs/openid-ca.pem` used here is the CA from the [generated TLS assets](https://github.com/dexidp/dex/blob/master/Documentation/kubernetes.md#generate-tls-assets), and is assumed to be present on the cluster nodes.
```

Comment:
Could you make that a relative link? I think `[generated TLS assets](#generate-tls-assets)` might do the trick here.